test/integration/targets/vmware_guest_facts/tasks/main.yml

```diff
@@ -40,8 +40,8 @@
       - "guest_facts_0001['instance']['guest_consolidation_needed'] is defined"
       - "guest_facts_0001['instance']['moid'] is defined"
       - "guest_facts_0001['instance']['vimref'] is defined"
-      - "'portgroup_portkey' in guest_facts_0001['instance']['hw_eth0']"
-      - "'portgroup_key' in guest_facts_0001['instance']['hw_eth0']"
+      - "'portgroup_portkey' in guest_facts_0001['instance']['Network adapter 1']"
+      - "'portgroup_key' in guest_facts_0001['instance']['Network adapter 1']"
```

Comment:
```suggestion
      - "'portgroup_key' in guest_facts_0001['instance']['Network Adapter 1']"
```

Reply:
Are you seeing in your testing that vmware is returning the vmware_guest instance with a capital A in in the entry.deviceInfo.label for each adapter?

Reply:
I think for govcsim (which is used for testing `vmware_guest` it is returning `Network Adapter 1`

```
(Pdb) [entry.deviceInfo.label for entry in vm.config.hardware.device if hasattr(entry, 'macAddress')]
['Network Adapter 1', 'ethernet--7']
```

Reply:
Oh well that explains why. Veri confusing. Why is govcsim modifying the return values of the VMware network labels? That doesn't seem like a good practice. Anyway, thanks for the info! I'll accept your changes soon as I'm back home.